src/growthbook.ts

```diff
@@ -11,7 +15,7 @@ export type AttributesTypes = {
     device_type: string
 }
 export class Growthbook {
-    GrowthBook
+    GrowthBook: GrowthBook
```

Comment:
```suggestion
    GrowthBook
```

I think we don't need to change this line.

Reply:
remove a type
and yes, this type wasn't helpful, thanks you for repeating the comment, I didn't get you first time